src/picknik_ur_site_config/objectives/open_door_fixed_handle_MTC_config.yaml

```diff
@@ -7,10 +7,9 @@ SetupMTCOpenDoorAffordance:
   end_effector_open_pose_name: "open"
 
   # List of controllers to use during the stages that interact with the door.
-  # For this objective, we want to enable admittance control.
-  door_opening_controllers: "/joint_trajectory_controller_chained_open_door /admittance_controller_open_door /robotiq_gripper_controller"
+  move_along_arc_controllers: "/joint_trajectory_controller_chained_open_door /admittance_controller_open_door /robotiq_gripper_controller"
```

Comment:
Make sure the Gazebo config overrides this file, as admittance control does not wotk in Gazebo.

Reply:
Should this just be an empty chain then? ""

Reply:
The update-admittance controller is wrapped in a force success to override this, however this should prevent one of those errors at the end, so added the file. 
Maybe there's a cleaner way to do all this though as these controllers are also defined (hardcoded) in the open_door objectives (without the door_admittance_controller).